tests/test_files_list.py

```diff
@@ -180,6 +180,88 @@ def test_list_files_auth(client):
     )
 
 
+def test_list_files_root_path(client):
+    """List files when the root path is requested. There can be 3 options: '/'  './' and '.'"""
+
+    # Make project available
+    response = client.post(
+        tests.DDSEndpoint.PROJECT_STATUS,
+        headers=tests.UserAuth(tests.USER_CREDENTIALS["unituser"]).token(client),
+        query_string={"project": "public_project_id"},
+        json={"new_status": "Available"},
+    )
+    assert response.status_code == http.HTTPStatus.OK
+
+    # "/" is the root path
+    response = client.get(
+        tests.DDSEndpoint.LIST_FILES,
+        headers=tests.UserAuth(tests.USER_CREDENTIALS["researchuser"]).token(client),
+        query_string={"project": "public_project_id"},
+        json={"subpath": "/"},
+    )
+    expected = {
+        "files_folders": [
+            {"folder": True, "name": "filename1"},
+            {"folder": True, "name": "filename2"},
+            {"folder": True, "name": "sub"},
+        ]
+    }
+    assert "files_folders" in response.json
+    assert len(response.json["files_folders"]) == len(expected["files_folders"])
+    for entry in response.json["files_folders"]:
+        assert len(entry) == 2
+        assert entry["folder"] is True
+    assert set(entry["name"] for entry in response.json["files_folders"]) == set(
+        entry["name"] for entry in expected["files_folders"]
+    )
```

Comment:
This section is replicated 3x times here so could make the test parameterised on the `subpath` (using https://docs.pytest.org/en/7.1.x/example/parametrize.html). 

Also, the section from `expected` to the asserts (L202-216), is identical here and in the `tests_v3` section so we could pull it out as a helper function in a separate file in the testing directory (something like _helpers.py_)?